### PR TITLE
docs: Add HowToConfigureMemory doc and integrate the memory guide in …

### PR DESCRIPTION
**Pull request by jeongchanKim**
…Porting_Guide

HowToConfigureMemory doc has information about memory.
HowToUseMultiHeap guide should be in HowToConfigureMemory doc.

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToConfigureMemory.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+CONFIG_RAM_START=0x02023800
+CONFIG_RAM_SIZE=968704
+```
+Heap allocation will use these values for setting heap start and end address.
+
+### Linker Scripts
+
+Linker script has RAM start and size information also.  
+file path : *build/configs/xxx_board/scripts/xxx.script*
```
Comment (sunghan-chang) [inline docs/HowToConfigureMemory.md]: file path : *build/configs/<BOARD_NAME>/scripts/<SCRIPT_NAME>.ld*

File: docs/HowToConfigureMemory.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# How to configure Memory
+
+## Contents
+> [Memory Configuration](#memory-configuration)  
+> [MultiHeap Support](#multiheap-support)
+
+## Memory Configuration
+For configuring memory, there are 2 points to change.  
+
+### TizenRT configuration file
```
Comment (sunghan-chang) [inline docs/HowToConfigureMemory.md]: Let's use "Configuration".

File: docs/HowToConfigureMemory.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+...
+	SRAM	(rwx)	: ORIGIN = 0x02023800, LENGTH = 946K
+...
+}
+```
+These value should be matched with TizenRT configuration file.
+
+## MultiHeap Support
+TizenRT can support multi-heap, if there is another RAM partition in H/W with discontinuous address to previous one.  
+[How to use multiple memory map as a heap](HowToUseMultiHeap.md)
```
Comment (sunghan-chang) [inline docs/HowToConfigureMemory.md]: Please refer [How to use multiple memory map as a heap](HowToUseMultiHeap.md).

**Comment by seinfra** [comment]
Code Rule Check OK.

File: docs/HowToConfigureMemory.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+# How to configure Memory
+
+## Contents
+> [Memory Configuration](#memory-configuration)  
+> [MultiHeap Support](#multiheap-support)
+
+## Memory Configuration
+For configuring memory, there are 2 points to change.  
+
+### TizenRT Configuration file
```
Comment (sunghan-chang) [inline docs/HowToConfigureMemory.md]:
```
### Configuration
There are two configurations to configure memory, CONFIG_RAM_START and CONFIG_RAM_SIZE.  
Menuconfig helps to change them as shown below:
!```
cd xxxx
make menuconfig
!```
Find configs with following step:
!```
xxxx
!```
Set CONFIG_RAM_START with?as? hexa value and set xx with?as? decimal
!```
CONFIG_RAM_START=0x19817983
CONFIG_RAM_SIZE=2398293
!```
Heap allocation will use these values for setting heap start and end address.

### Linker Scripts
```

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**